tests/phpunit/tests/test-admin-filters-term.php

```diff
@@ -178,6 +185,168 @@ public function test_create_term_from_post_bulk_edit() {
 		$this->assertEqualSets( array( $new_fr, $test_fr, $third ), $tags );
 	}
 
+	/**
+	 * @ticket #1766 see {https://github.com/polylang/polylang-pro/issues/1766}.
+	 */
+	public function test_terms_translation_on_post_save() {
```

Comment:
I'd rather split the test, at least one for categories and another for tags (you could still reuse some code between tests).
Also I wonder if the test belongs to `Admin_Filters_Term_Test` since it's not only related to admin.

Reply:
Moved to `CRUD_Posts_Test` in [1259983](https://github.com/polylang/polylang/pull/1336/commits/125998325cdcd4f7cc7934a0fd45294bce8364cb).
However, what would be the benefits of splitting it?

Reply:
Done in [dd4b6c1](https://github.com/polylang/polylang/pull/1336/commits/dd4b6c11d718741d74821b029c88d1ab045e05b1).